app/config_testing.go

```diff
@@ -77,6 +76,8 @@ func setBaseDenom(ci evmtypes.EvmCoinInfo) (err error) {
 	if err := sdk.RegisterDenom(ci.DisplayDenom, math.LegacyOneDec()); err != nil {
 		return err
 	}
+
+	// new setBaseDenom() are lower than the current base denom's units.
 	// sdk.RegisterDenom will automatically overwrite the base denom when the new denom units are lower than the current base denom's units.
```

Comment:
nit: the comment is not ordered correctly


```suggestion

	// sdk.RegisterDenom will automatically overwrite the base denom when the
	// new setBaseDenom() units are lower than the current base denom's units.
```